src/internal_events/tag_cardinality_limit.rs

```diff
@@ -42,7 +42,7 @@ pub struct TagCardinalityValueLimitReached<'a> {
 impl<'a> InternalEvent for TagCardinalityValueLimitReached<'a> {
     fn emit(self) {
         debug!(
-            "Value_limit reached for key {}. New values for this key will be rejected.",
+            "Value_limit reached for key {key}. New values for this key will be rejected.",
```

Comment:
```suggestion
            "Value_limit reached for key. New values for this key will be rejected.",
```

Per https://github.com/vectordotdev/vector/blob/master/docs/DEVELOPING.md#logging-style we generally try to avoid templating values into log strings to make them easier to match.

Reply:
Fixed.